testing/sdk_install.py

```diff
@@ -90,10 +90,6 @@ def install(
         insert_strict_options=True):
     start = time.time()
 
-    # If the package is already installed at this point, fail immediately.
```

Comment:
I'm not sure we should be changing the copied `testing/` subdirectory in the same PR as changing local code.